storefront/test/integration/workarea/storefront/recaptcha_integration_test.rb

```diff
@@ -0,0 +1,133 @@
+require 'test_helper'
+
+module Workarea
+  module Storefront
+    class RecaptchaIntegrationTest < Workarea::IntegrationTest
+      setup :enable_recaptcha
+      teardown :disable_recaptcha
+
+      def enable_recaptcha
+        ::Recaptcha.configuration.skip_verify_env.delete('test')
+      end
+
+      def disable_recaptcha
+        ::Recaptcha.configuration.skip_verify_env << 'test'
+      end
+
+      def setup_v3
+        Workarea.config.recaptcha_v3_site_key = '6LdxWf0ZAAAAAPb40p0---l92ifZiWbNOPX6ujYv'
+        Workarea.config.recaptcha_v3_secret_key = '6LdxWf0ZAAAAAPsETRjMxjmv2NEKjvJQ6xrhaxLB'
```

Comment:
are these keys that are ok to commit? like official test keys or something?

Reply:
Just some test ones I generated for cassette recording, but may as well update to fake ones I guess.